aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -72,11 +68,11 @@ const allData = [
   },
 ];
 
-// Define which attributes should be made available for the user
-// to filter upon
+// Define which attributes are available for the user to filter upon
 const filtersConfig = [
-  { property: 'role', label: 'Role', filterType: 'CheckBoxGroup' },
+  { property: 'name', label: 'Name', filterType: 'CheckBoxGroup' },
```

Comment:
I don't think "filterType" applies anymore since you're using Data. Can we remove?

Reply:
Thanks for catching that!